src/gui/dialogs/multiplayer/mp_create_game.cpp

```diff
@@ -319,6 +320,15 @@ void mp_create_game::pre_show(window& win)
 		observers_->widget_set_enabled(win, false, false);
 		strict_sync_->widget_set_enabled(win, false, false);
 	}
+	// Disallow unregistered users if on official server
+	else if(official_server_) {
+		registered_users_->widget_set_enabled(win, false, false);
+		registered_users_->set_widget_value(win, true);
```

Comment:
This enables registered users only? Why set it to true? Since everyone is registered there is no need to check so rather turn it off.

Reply:
As per the comment in https://github.com/wesnoth/wesnoth/issues/2939#issuecomment-531472160

This checks the box and disables it from being changed.

Reply:
IIRC DUL can be disabled server side with games already running using `/query`, so I don't think it's an issue to have the client enforce registered users only on created games to avoid unexpected surprises if for whatever reason the option is disabled without restarting the server.

Reply:
Registered users only should not be enforced so it works just fine if we turn DUL off for whatever reason. Forcing it to on does not help anything if DUL is on and can only prevent playing if DUL is off because the DB is down or whatever and no one is registered.